src/components/pod/pod.style.ts

```diff
@@ -1,36 +1,52 @@
 import styled, { css } from "styled-components";
-import { margin } from "styled-system";
+import { margin, MarginProps } from "styled-system";
 
 import { baseTheme } from "../../style/themes";
 import Link from "../link";
 import { StyledContent as StyledLinkContent } from "../link/link.style";
 import IconButton from "../icon-button";
 import StyledIcon from "../icon/icon.style";
-
-const StyledPod = styled.div`
+import { PodAlignment, PodSize, PodVariant } from "./pod.config";
+
+export interface StyledPodProps extends MarginProps {
+  alignTitle: PodAlignment;
+  internalEditButton?: boolean;
+  height?: string;
+}
+const StyledPod = styled.div<StyledPodProps>`
+  ${margin}
   display: flex;
   align-items: flex-start;
   width: 100%;
   text-align: ${({ alignTitle }) => alignTitle};
   ${({ internalEditButton }) => internalEditButton && "position: relative"};
-  ${margin}
   ${({ height }) => height && `height: ${height}`};
 
   &:focus {
     outline: none;
   }
 `;
 
-const blockBackgrounds = (variant) =>
-  ({
-    primary: "var(--colorsUtilityYang100)",
-    secondary: "var(--colorsUtilityMajor025)",
-    tertiary: "var(--colorsUtilityMajor040)",
-    transparent: "var(--colorsUtilityMajorTransparent)",
-    tile: "var(--colorsUtilityYang100)",
-  }[variant]);
+const blockBackgrounds = {
+  primary: "var(--colorsUtilityYang100)",
+  secondary: "var(--colorsUtilityMajor025)",
+  tertiary: "var(--colorsUtilityMajor040)",
+  transparent: "var(--colorsUtilityMajorTransparent)",
+  tile: "var(--colorsUtilityYang100)",
+};
 
-const StyledBlock = styled.div`
+export interface StyledBlockProps {
+  variant: PodVariant;
+  noBorder: boolean;
+  softDelete?: boolean;
+  hasButtons: boolean;
+  contentTriggersEdit?: boolean;
+  fullWidth?: boolean;
+  internalEditButton?: boolean;
+  isHovered?: boolean;
+  isFocused?: boolean;
+}
```

Comment:
I think that we could pick most of this props out of the main interface.
What do you think of changing "fullWidth" to "editContentFullWidth" to be consistent?

Reply:
Thanks for spotting this @grabkowski, seems I forgot to go back and amend these after we decided on this. Will amend now 👍🏼